beacon-chain/p2p/broadcaster.go

```diff
@@ -202,6 +202,38 @@ func (s *Service) broadcastSyncCommittee(ctx context.Context, subnet uint64, sMs
 	}
 }
 
+// BroadcastBlob broadcasts a blob to the p2p network, the message is assumed to be
+// broadcasted to the current fork and to the input subnet.
+func (s *Service) BroadcastBlob(ctx context.Context, subnet uint64, blob *ethpb.SignedBlobSidecar) error {
+	ctx, span := trace.StartSpan(ctx, "p2p.BroadcastBlob")
+	defer span.End()
+	if blob == nil {
```

Comment:
is this the only validation we need to do before a blob sidecar is broadcasted? do we need to validate blob with kzg?

Reply:
This is just broadcast from p2p's perspective. Validations (if there are any) are done before this